rope/refactor/extract.py

```diff
@@ -34,7 +35,7 @@
 # There are a few more helper functions and classes used by above
 # classes.
 class _ExtractRefactoring:
-    kind_prefixes = {}
+    kind_prefixes: Dict[str, str] = {}
```

Comment:
To be more accurate, this could've used Literal to define that it doesn't accept arbitrary string, just a small enumeration of it. Literal is only for Python 3.8 and above though, so we can't use it yet.

Reply:
@lieryan I had forgotten about `Literal`, but adding it seems like small beer.